app/models/post.rb

```diff
@@ -1,2 +1,15 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, foreign_key: 'post_id'
+  has_many :likes, foreign_key: 'post_id'
+
+  after_create :update_posts_counter
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def recent_comments
+    comments.limit(5)
+  end
```

Comment:
- [ ] Again here we also only want the most recent comments. Kindly implement an order fetch.

  ```
  def recent_comments
    comments.order('created_at Desc').limit(5)
  end
  ```

Reply:
well resolved! ✔️